.github/workflows/windows-pr.yml

```diff
@@ -37,13 +37,17 @@ jobs:
       with:
         go-version: ${{ matrix.go-version }}
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
     - name: Install Python Deps
       run: |
         pip3 install pyenv-win
         pip3 install pipenv
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
```

Comment:
Is this for debugging or you want both of these two steps to stay?

Reply:
Sorry, yes, I should remove that!! That was indeed for debugging